DataWorkerService/Services/Implementations/OsuApiDataParserService.cs

```diff
@@ -38,8 +38,6 @@ public async Task ParseMatchAsync(Match match, MultiplayerMatch apiMatch)
             match.Games.Add(newGame);
         }
 
-        match.EndTime = DetermineMatchEndTime(apiMatch) ?? default;
```

Comment:
Why was this removed? No match end times will be populated if this is removed.

Reply:
I think I deleted this by mistake when I removed some of the code I had here.